c2corg_ui/static/menu-discourse/README.md

```diff
@@ -0,0 +1,35 @@
+
+### Menu on the Discourse Forum
+
+The goal was to rebuild the exact same menu in both UI and on the Discourse Forum.
+Only a few icons change (to differenciate between the already-too-similar-ones on Discourse) and the HTML for the menu on Discourse had to be adapted.
+
+If you make any changes to the `sidemenu.html` or `sidemenu.less`, make sure you update the Discourse's menu too.
+
+The HTML changes will have to be adapted manually.
+
+The less file (`sidemenu.less`) was written to suit both the UI and Discourse, even though we won't use the exact same lines in both cases, but the code stays mostly the same.
+
+The files in Discourse should be changed under **the hamburger button** -> **admin** -> **customize (personnaliser)** -> **CSS/HTML** -> and choose from the left list **discourse menu** or create a new style.
+
+
+## LESS -> CSS
+In order to compile the usual sidemenu.less file into pure CSS, use the command:
+
+`./node_modules/.bin/lessc less/sidemenu.less  > sidemenu.css`
```

Comment:
> you mean you prefer to add an additional activity to make less that will output this?

Yes

For me it's not consistent to use the way you suggest: all styles files should be built the same way and with the same command.
If you have made changes to the sidemenu, it means that the makefile will have to be run for the UI anyway. Than the forum admin can simply get the forum menu styles by opening  http://www.demov6.camptocamp.org/static/4120b0e/build/build-forum-menu.min.css (or another name) in their browser and copy-paste it in the admin interface (or perhaps it will be possible in the futur to simply link the CSS file).

Your way is more complicated, you have to find what command to run, then run it (and you need to have the environnement to run it, with up-to-date code etc.).

Using the makefile is not difficult to implement and way more consistent/convenient.

Please do use the makefile.